src/components/loading/loading.js

```diff
@@ -1,6 +1,10 @@
-define(['css!./loading-lite'], function () {
+define(['components/loading/loadingLegacy', 'browser', 'css!./loading'], function (loadingLegacy, browser) {
     'use strict';
 
+    if (browser.tizen || browser.operaTv || browser.chromecast || browser.orsay || browser.web0s || browser.ps4) {
```

Comment:
care to explain the change?

Reply:
This was moved from site.js to loading.js. I merged loading components into one, so webpack could resolve this into one single loading module.